tox.ini

```
@@ -8,6 +8,7 @@ setenv =
 deps = 
     django32: Django>=3.2,<4.0
     -r requirements/test.txt
+    django42: Django>=4.2,<4.3
 commands = 
     pytest --cov=django_sites_extensions
 
```

Let's try including this line to fix coverage issue.
```suggestion
        coverage xml
```